arduino/smartcar/smartcar.ino

```diff
@@ -242,6 +308,7 @@ void reactToSides() {
         }
     }
 }
+
```

Comment:
in the sideavoidance(), there are possible data loss from int getDistance() to float.